pagerduty provider v1.6.1 seems broken

Here is the error log I have got (which works for v1.5.1)

```
panic: runtime error: invalid memory address or nil pointer dereference
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd326cb]
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: 
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: goroutine 59 [running]:
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: github.com/terraform-providers/terraform-provider-pagerduty/pagerduty.resourcePagerDutyServiceIntegrationRead.func1(0xc0002ee500)
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-pagerduty/pagerduty/resource_pagerduty_service_integration.go:147 +0xdb
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: github.com/hashicorp/terraform-plugin-sdk/helper/resource.Retry.func1(0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-pagerduty/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/resource/wait.go:22 +0x4e
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: github.com/hashicorp/terraform-plugin-sdk/helper/resource.(*StateChangeConf).WaitForState.func1(0xc0004293e0, 0xc0002ccb60, 0xc0000c6300, 0xc000041c20, 0xc000099600, 0xc0000995f8)
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-pagerduty/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/resource/state.go:103 +0x29e
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: created by github.com/hashicorp/terraform-plugin-sdk/helper/resource.(*StateChangeConf).WaitForState
2020-04-14T20:05:56.008Z [DEBUG] plugin.terraform-provider-pagerduty_v1.6.1_x4: 	/opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-pagerduty/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/resource/state.go:80 +0x1b9
2020-04-14T20:05:56.009Z [DEBUG] plugin: plugin process exited: path=/home/travis/build/meetup/meetup-pagerduty-config/.terraform/plugins/linux_amd64/terraform-provider-pagerduty_v1.6.1_x4 pid=3348 error="exit status 2"
2020/04/14 20:05:56 [ERROR] <root>: eval: *terraform.EvalRefresh, err: rpc error: code = Unavailable desc = transport is closing
2020/04/14 20:05:56 [ERROR] <root>: eval: *terraform.EvalSequence, err: rpc error: code = Unavailable desc = transport is closing
```

--- a/pagerduty/resource_pagerduty_service_integration.go
+++ b/pagerduty/resource_pagerduty_service_integration.go
@@ -28,6 +28,7 @@ func resourcePagerDutyServiceIntegration() *schema.Resource {
 			"service": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
 			},
 			"type": {
 				Type:          schema.TypeString,
@@ -142,6 +143,8 @@ func resourcePagerDutyServiceIntegrationRead(d *schema.ResourceData, meta interf
 				log.Printf("[WARN] Returning retryable error")
 				return resource.RetryableError(errResp)
 			}
+
+			return nil
 		}
 
 		d.Set("name", serviceIntegration.Name)

